pkg/reconcilers/dummy_policy_reconciler.go

```diff
@@ -0,0 +1,59 @@
+package reconcilers
+
+import (
+	"context"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type DummyPolicy struct {
+	client.Object
+
+	Status struct {
+		Conditions []metav1.Condition
+	}
+}
+
+func (p *DummyPolicy) GetStatusConditions() *[]metav1.Condition {
+	return &p.Status.Conditions
+}
+
+type DummyPolicyReconciler struct {
```

Comment:
An example of how a reconciler would look like using the generic policy reconciler